packages/react-server-cli/package.json

```diff
@@ -14,7 +14,7 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "babel-core": "~6.5.1",
+    "babel-core": "^6.9.1",
```

Comment:
These version changes resolve [nsp 106](https://nodesecurity.io/advisories/106), but we can't remove the exception from generator-react-server in this pr due to [this nsp issue](https://github.com/nodesecurity/nsp/issues/117)